dist/stat_data.py

```diff
@@ -452,6 +452,7 @@ def __init__(self, name, desc, flags=''):
     YieldStat('page_locked_blocked', 'page acquire locked blocked'),
     YieldStat('page_read_blocked', 'page acquire read blocked'),
     YieldStat('page_sleep', 'page acquire time sleeping (usecs)'),
+    YieldStat('txn_release_blocked', 'transaction release blocked shutdown'),
```

Comment:
I prefer the text "connection close blocked waiting for transaction state stabilization"